src/components/KMultiselect/KMultiselect.vue

```diff
@@ -828,6 +833,10 @@ watch(() => props.items, (newValue, oldValue) => {
       unfilteredItems.value[i].selected = false
     }
 
+    if (unfilteredItems.value[i].selected === true && unfilteredItems.value[i].disabled === true && !unfilteredItems.value[i].disabledTooltipText) {
```

Comment:
suggestion: adding a comment
```suggestion
    // If an item is `selected` and `disabled`, provide fallback tooltip text if not provided 
    if (unfilteredItems.value[i].selected === true && unfilteredItems.value[i].disabled === true && !unfilteredItems.value[i].disabledTooltipText) {
```